cypress/integration/page.spec.js

```diff
@@ -455,7 +484,7 @@
     describe('on posts loading error', () => {
       beforeEach(() => {
         page.mockUsers();
-        page.mockError('**/posts?userId=1', 'user1PostsRequest')
+        page.mockError('**/posts?userId=1', 'user1PostsRequest');
```

Comment:
There is a missing semicolon at the end of the `cy.intercept` statement. Ensure that each statement is properly terminated to avoid syntax errors.